### ISSUE TITLE
Button or a link

### ISSUE THREAD

**Issue by iadawn** [ministryofjustice/record-a-goose-sighting#13]
On the first page the following issue is identified:

> `Role=“button”` missing on link to next page Visually, this is coded to look like a button, but it's actually a link. Links can only be selected by pressing enter, but buttons can be selected by pressing space bar too. This can be told visually, from it's appearance, and giving it a `role="button"` would let screen reader users know this too. This has come about from the Design System not being implemented properly, as the Design System markup contains this. This is a relatively minor issue, but it is best practise to add in appropriate ARIA if you're doing things like this.

I would question two things; why an `a` element semantic meaning is being modified with `role` and why this is considered a button at all.

Keyboard interactions for buttons are slightly more than those for links; buttons have `space` for activation as well as `enter` and links have the option for context menu with `shift + F10`. Conflating the two does risk confusion.

Basically links go places and buttons do things. This is an element that goes places. It doesn't have any functional thing associated with it so arguably it should be a link with all link semantics. There was a recent [CSS Tricks](https://css-tricks.com/a-complete-guide-to-links-and-buttons/) article that went into this in-depth.

**Comment by BNewing** [comment]
That is a good question! I am unsure of why it is that way, as it is a link and not a button, and it's just been styled to look like one. 

I took this from the [GOV.UK Design System, from the 'start button' component.](https://design-system.service.gov.uk/components/button/#start-buttons), which I understood had gone through a lot of testing and so I've taken it on faith. 

One reason I'd understood it might be is that it's to try to marry up information in the accessibility tree with what's visually presented - but then I'm not sure how much sense that makes, on reflection! I will ask :)

**Comment by BNewing** [comment]
I've had a chat about this, and it sounds like the benefit might be for users of Dragon, but that it's acknowledged that it's a little controversial, for the reasons you've outlined.

It's something that rarely (if ever) gets spotted by people. I originally included it because one issue I see happening a lot was Design System markup not being fully copied across. Some services can't directly consume it, and do have to copy and paste markup. The point was to draw people's attention to that you have to be careful, and check that it's still up to date.

But, given this one is quite complicated, and is a little tricky to explain, I'm thinking maybe it's best to remove it for now, and have a think about a simpler example to illustrate this point. What do you think @iadawn?

**Comment by iadawn** [comment]
Yeah, given that the goal is for anyone with some knowledge could pick this resource up and run with it, it might be better to have something simpler. It is a good point to illustrate!

**Comment by BNewing** [comment]
It was originally created for developers, and it made sense back at the start. It does feel a little too in-depth though, now. I'll create a PR and remove it :)

**Comment by jbuller** [comment]
I was going to suggest that you could leave it in but not list it in the 'simple' answers, leaving the option open for more 'advanced' answers.

**Comment by BNewing** [comment]
Sounds like a good idea :) could you raise it in a new issue please?
